tests/README.md

```diff
@@ -201,7 +206,7 @@ For executing this test, first make sure you are in the correct environment as d
 pytest tests/unit/test_notebooks_python.py::test_sar_single_node_runs
 ```
 
-### Developing smoke and integration tests with Papermill
+#### Developing smoke and integration tests with Papermill
```

Comment:
Here we need to update the code of the smoke test with SAR with scrapbook: https://github.com/microsoft/recommenders/blob/main/tests/smoke/examples/test_notebooks_python.py#L18

Reply:
Done